client/jetpack-cloud/sections/comparison/table/links.ts

```diff
@@ -21,11 +23,11 @@ export const links = {
 	payments_block: 'https://jetpack.com/support/jetpack-blocks/payments-block/',
 	priority_support: 'https://jetpack.com/features/security/expert-priority-support/',
 	related_posts: 'https://jetpack.com/features/traffic/related-posts/',
-	scan: 'https://jetpack.com/upgrade/scan/',
-	search: 'https://jetpack.com/upgrade/search/',
+	scan: localizeUrl( 'https://jetpack.com/upgrade/scan/' ),
+	search: localizeUrl( 'https://jetpack.com/upgrade/search/' ),
 	secure_authentication: 'https://jetpack.com/features/security/secure-authentication/',
```

Comment:
Just curious why we are localizing some but not all of these links for jetpack.com? Are some of the other pages not reliably translated?

Reply:
Yes, I've decided to localize product URLs which are translated into all languages. Feature or support pages are unfortunately not consistent in this.